src/AppBundle/Repository/CommitteeMembershipRepository.php

```diff
@@ -302,20 +302,41 @@ private function findPriviledgedMembers(string $committeeUuid, array $privileges
      */
     public function findMembers(string $committeeUuid): AdherentCollection
     {
+        return $this->createAdherentCollection($this->createCommitteeMembershipsQueryBuilder($committeeUuid)->getQuery());
+    }
+
+    /**
+     * Returns the list of all committee memberships of a committee.
+     *
+     * @param string $committeeUuid The committee UUID
+     *
+     * @return CommitteeMembershipCollection
+     */
+    public function findCommitteeMemberships(string $committeeUuid): CommitteeMembershipCollection
+    {
+        $query = $this
+            ->createCommitteeMembershipsQueryBuilder($committeeUuid)
+            ->addSelect('a')
+            ->getQuery()
+        ;
+
+        return new CommitteeMembershipCollection($query->getResult());
+    }
+
+    private function createCommitteeMembershipsQueryBuilder(string $committeeUuid, string $alias = 'cm'): QueryBuilder
```

Comment:
I agree with @mykiwi : using docblock is not useful with typehint, espacially for private methods.